packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -456,11 +452,72 @@ class HeapTreeViewState extends State<HeapTree>
   }
 
   Widget buildSnapshotTables(Widget snapshotDisplay) {
+    final themeData = Theme.of(context);
+
+    if (snapshotDisplay == null) {
+      // Display help text about how to collect data.
+      return Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: const [
+              Text('Click the take heap snapshot button '),
+              Icon(Icons.camera),
+              Text(' to collect a graph of memory objects.'),
+            ],
+          ),
+        ],
+      );
+    }
+
     final rightSideTable = controller.isLeafSelected
         ? InstanceTreeView()
         : controller.isAnalysisLeafSelected
             ? Expanded(child: AnalysisInstanceViewTable())
-            : const SizedBox();
+            : Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  const Text('Click a leaf node instance of a class,'),
```

Comment:
remove comma after class, and instead of creating a new text widget below, just add a new line char in the middle of the text for this text widget

Reply:
Did a bunch more cleanup around the code.